validated-method.js

```diff
@@ -17,14 +17,32 @@ ValidatedMethod = class ValidatedMethod {
       options.validate = function () {};
     }
 
+    if (options.callOptions === undefined) {
+      options.callOptions = {};
+    }
+
     check(options, Match.ObjectIncluding({
       name: String,
       validate: Function,
       run: Function,
       mixins: [Function],
       connection: Object,
+      callOptions: Object,
     }));
 
+    const defaultCallOptions = {
+      // Make it possible to get the ID of an inserted item
+      returnStubValue: true,
+
+      // Don't call the server method if the client stub throws an error, so that we don't end
+      // up doing validations twice
+      // XXX needs option to disable, in cases where the client might have incomplete information to
+      // make a decision
+      throwStubExceptions: true,
+    };
+
+    options.callOptions = _.extend(defaultCallOptions, options.callOptions);
```

Comment:
Hey, I think this will actually overwrite the `defaultCallOptions` with `options.callOptions`. So we probably want:

```
_.extend({}, defaultCallOptions, options.callOptions);
```